test-suite/js/src/test/scala/org/scalajs/testsuite/library/ObjectTest.scala

```diff
@@ -97,6 +97,9 @@ class ObjectTest {
   }
 
   @Test def fromEntries_js_Map(): Unit = {
+    assumeTrue("requires js.Map",
+        js.typeOf(js.constructorOf[js.Map[_, _]]) != "undefined")
```

Comment:
Put a forwarder for this in `org.scalajs.testsuite.utils.Platform` and force it to be true if `assumeES2015` is true?

Reply:
Done.